### PR TITLE
Scaffold: Animation

### PR DESCRIPTION
**Pull request by varffs**
- enqueues images from site option
- triggers load of x images on init
- loads all additional images with x delay
- animates with timeout (rAF not designed for fixed framerate animation)

Works as I think this should work. Seems reasonably smooth (tho blank starting sucks on slower connection perhaps there can be a spinner) Performance doesnt seem to be great not sure how to improve that.

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -29,6 +30,103 @@ Site = {
...
+      if ($next.length) {
+        $next.addClass('active');
+      } else {
+        $('.animation-image-loaded').first().addClass('active');
+      }
+    }, _this.animationSpeed);
+
+  },
+
+  updateAnimation: function() {}
```
Comment (mikeyrayvon) [inline src/js/main.js]: what goes here?

Comment (mikeyrayvon) [reply src/js/main.js]: i guess you were going to use this func and then no?

**Review by mikeyrayvon — COMMENTED** [review]
I'm good with this. appears to work ok. may need a slightly different markup strategy to get the images to absolute center... unless we're going to full height with them? i'm ok to merge unless @mixcas has an issue

File: src/js/main.js (hunk 1 above)
Comment (varffs) [reply src/js/main.js]: oh yh that is probably dead code i will remove

**Comment by mixcas** [comment]
TBH I don't like the code, it's hard to read and follow, doesn't have any comments, theres repeated code  and `console.log('all loaded')` gets called multiple times which mean the code is doing something that's not intended or the message is wrong or i'm just not understanding anything at all. I just wanted to bring this up because to review this I had to debug the code step by step on DevTools. 

It works fine and it doesn't really require any early optimization so I'm not gonna get against it and waste time and merge it.

**Comment by varffs** [comment]
Yh def I should have commented sorry. I'll add that as a feature then at that moment perhaps some suggestions would come to light at that moment. If you are not happy with the code I don't want to ship it like that.

**Pull request merged**